Makefile

```diff
@@ -5,6 +5,7 @@ all:
 	cd cloud && $(MAKE)
 	cd edge && $(MAKE)
 	cd keadm && $(MAKE)
+	cd device/bluetooth_mapper && $(MAKE)
```

Comment:
please use $(MAKE) -C device/bluetooth_mapper instead of cd device/bluetooth_mapper && $(MAKE)

Reply:
@subpathdev , I gave MAKE command like that so as maintain consistency with rest of commands...Thoughts please....cc @rohitsardesai83 , @sids-b

Reply:
Yess... lets keep it consistent

Reply:
Updated